src/lib/artefact.ml

```diff
@@ -129,7 +129,7 @@ let () =
         )
       |})
 
-let get ~st ~judge ~target ~descr =
+let get ~st ~(judge:Judge.id) ~(target:Dancer.id) ~descr =
```

Comment:
```suggestion
let get ~st ~(judge:Judge.id) ~(target: Heat.target_id) ~descr =
```

The target here is not the id of a dancer, but a Heat target it (basically a row id from the heats table), so that it can refer to a single dancer for jack and jills, but also a couples for strictlys.